File: .sdk/tm/php/utility/ResultBody.php
<?php
declare(strict_types=1);

// Hook0 SDK utility: result_body

class Hook0ResultBody
{
    public static function call(Hook0Context $ctx): ?Hook0Result
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: .sdk/tm/csharp/utility/ResultBody.cs
// Hook0 SDK utility: resultBody.

namespace Hook0Sdk.Util;

public static partial class SdkUtility
{
    internal static Result ResultBodyUtil(Context ctx)
    {
        var response = ctx.Response;
        var result = ctx.Result;

        if (result != null)
        {
            if (response?.JsonFunc != null && response.Body != null)
            {
                result.Body = response.JsonFunc();
            }
        }

        return result!;
    }
}
